_pages/en_US/installing-boot9strap-(usm).txt

```diff
@@ -23,40 +23,23 @@ If your (Right/Left Shoulder), (D-Pad Up), or (A) buttons do not work, you will
 
 ### What You Need
 
-* Your `movable.sed` file from completing [Seedminer](seedminer)
 * The latest release of [SafeB9SInstaller](https://github.com/d0k3/SafeB9SInstaller/releases/download/v0.0.7/SafeB9SInstaller-20170605-122940.zip) (direct download)
 * The latest release of [boot9strap](https://github.com/SciresM/boot9strap/releases/download/1.4/boot9strap-1.4.zip) (direct download)
 * The latest release of [Luma3DS](https://github.com/LumaTeam/Luma3DS/releases/latest) (the Luma3DS `.zip` file)
+* The latest release of [unSAFE_MODE](https://github.com/zoogie/unSAFE_MODE/releases/latest) (the `usm.bin` file)
```

Comment:
since it's only being used on this page, why not just direct link to usm.bin

Reply:
Resolved.